ember.md

```diff
@@ -1,30 +1,28 @@
 # Ember
 
 1. **[General Structure](#general-structure)** 
-	[ [Properties](#general-structure--properties) ]
-	[ [Multi-property](#14-multi-property-components) ]
-  [ [Naming Addons](#general-structure--naming-addons) ]
+	[ [Properties](#1.1-Model-Property-Order) ]
+  [ [Naming Addons](#2.0-Naming-Addons) ]
 1. **[Destructuring](#destructuring)** 
```

Comment:
You'll need to remove the decimal points for these to work.

Reply:
Good catch! This worked in VSC preview but not Github. Fixed.